Apps/Examples/Examples/All Examples/CameraAnimationExample.swift

```diff
@@ -19,7 +19,7 @@ public class CameraAnimationExample: UIViewController, ExampleProtocol {
 
             // Center the map camera over New York City.
             let centerCoordinate = CLLocationCoordinate2D(
-                latitude: 40.7128, longitude: -74.0060)
+                latitude: 140.7128, longitude: -74.0060)
```

Comment:
Was that change intentional?

Reply:
Yes, to trigger a crash, will revert it when everything goes well